frontend/src/hooks/useCallbackPrompt.tsx

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+/* eslint-disable  @typescript-eslint/no-explicit-any */
```

Comment:
i think we need to avoid comments like that, maybe we could change code for delete ESlint comments?

Reply:
No, as I already mentioned in Overview this hook replace React router v.5 functionality, which is not supported in v6, so there is no Location types for it, so it was needed to use 'any' type and this comment helps to avoid eslint warnings in console and terminal

Reply:
ok, got it